web-app/HeimScripts/heatmap/run.R

```diff
@@ -393,3 +484,40 @@ addClusteringOutput <- function(jsn, measurements) {
   )
   return(jsn)
 }
+
+### Limma part. It will be moved into a separate
+# module when we get R scripts sourcing to work here.
+
+getDEgenes <- function(df) {
```

Comment:
Are there additional constraints for this (additional to at least 2 samples and at least 2 subsets), for example at least 2 sample per subset or something?
What if there are NAs? 
(I haven't looked into the details of this function yet)